lib/appoptics_apm/support/transaction_settings.rb

```diff
@@ -65,6 +66,14 @@ def to_s
     end
 
     private
+
+    ##
+    # check the config setting for :tracing_mode
+    def tracing_mode_disabled?
+      AppOpticsAPM::Config[:tracing_mode] &&
+        [:disabled, :never].include?(AppOpticsAPM::Config[:tracing_mode].to_sym)
```

Comment:
i don't know if this is called a lot but, if it is, it might make sense to keep AppOpticsAPM::Config[:tracing_mode] as a symbol so it needn't be converted each call.

Reply:
reactivated the code that actually did that ;)